src/styles/index.scss

```diff
@@ -1,3 +1,100 @@
+@import 'fonts';
+@import 'stars';
+@import 'variables';
+
 body {
   margin: 0;
+  padding: 0;
+  font-family: $fontRoboto;
+  font-size: 12px;
+  box-sizing: border-box;
+}
+
+.product-card {
+  box-sizing: border-box;
+  width: 200px;
+  height: 408px;
+  padding: 16px;
+  padding-top: 32px;
+  background-color: $basicWhite;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  &__img {
+    width: 160px;
+    max-width: 100%;
+    height: 135px;
+    margin-bottom: 40px;
+  }
```

Comment:
The `max-width` property on line 29 is unnecessary because you have already set the width on line 28. The `max-width` property is used to set the maximum width of an element. It prevents the used value of the `width` property from becoming larger than the value specified by `max-width`. But in your case, the width will never exceed 160px, so you can safely remove the `max-width` property.